cegs_portal/search/static/search/js/experiment_list/drag_drop.js

```diff
@@ -116,15 +116,29 @@ function experimentListItemText(name, accession) {
     return `<span class="experiment-list-item" data-accession="${accession}"><span class="exp-name">${name}</span> <span class="accession-id">${accession}</span></span>`;
 }
 
+function selectAllExperiments() {
+    let selectExperiments = document.getElementsByClassName("select-experiment");
+    for (let experiment of selectExperiments) {
+        experiment.click();
+        addToExperimentList(experimentListItemText(experiment.dataset.name, experiment.dataset.accession));
```

Comment:
Do you need both of these? Or will just one work?

Reply:
Need both to work correctly

Reply:
Are you _sure_? What have you tried?

Reply:
Are you referring to 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/21b05128-9d2b-44bc-86af-159539bee402)

Reply:
I'm referring to the two lines I highlighted